x/ccv/provider/migrations/v6/legacy_params.go

```diff
@@ -88,5 +88,7 @@ func GetParamsLegacy(ctx sdk.Context, paramspace ccvtypes.LegacyParamSubspace) t
 		getSlashMeterReplenishFraction(ctx, paramspace),
 		getConsumerRewardDenomRegistrationFee(ctx, paramspace),
 		getBlocksPerEpoch(ctx, paramspace),
+		// this parameter is new so it doesn't need to be migrated, just initialized
```

Comment:
It seems to me that `MaxProviderConsensusValidators` will be introduced after the `v6` migration, so it's not part of the legacy params.

Reply:
I will make it a separate function to initialize this